jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/Constants.java

```diff
@@ -0,0 +1,10 @@
+package io.opentelemetry.contrib.jfr;
+
+public class Constants {
```

Comment:
If what I said about exposing a programmatic entrypoint makes sense, then the API matters so let's only expose `public` what's needed (I'm guessing just the one entrypoint class)